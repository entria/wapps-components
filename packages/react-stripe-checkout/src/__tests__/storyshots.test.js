import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  storyKindRegex: /^react-stripe-checkout|(StripeCheckout)$/,
});
