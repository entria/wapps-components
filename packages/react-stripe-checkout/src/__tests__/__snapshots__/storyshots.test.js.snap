// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots react-stripe-checkout|StripeCheckout default 1`] = `
<button
  onClick={[Function]}
>
  Purchase
</button>
`;

exports[`Storyshots react-stripe-checkout|StripeCheckout with custom button 1`] = `
<button
  onClick={[Function]}
>
  Custom
</button>
`;
